### ISSUE TITLE
Strange Syntax Error

### ISSUE THREAD

**Issue by jwardle** [CocoaPods/CocoaPods#4299]
Hi, 

I am receiving the below error reported by Cocoapods when performing 'pod update' in response to a certain line. This project has not changed, nor has the podfile in a few months (I know, as I was the only one working on it) and Cocoapods version has not changed. When coming back to the project and running a standard 'pod update' or install,  it results in the syntax reader choking on the following line:

    # Logging
    pod 'CocoaLumberjack', :git => 'https://github.com/CocoaLumberjack/CocoaLumberjack.git', :branch => 'swift_2.0'

The error reported by Cocoapods on this line is: 
    $ pod install
    
    [!] Invalid `Podfile` file: syntax error, unexpected tSTRING_BEG, expecting end-of-input
    ...Lumberjack/CocoaLumberjack.git', :branch => 'swift_2.0'
    ...                               ^. Updating CocoaPods might fix the issue.

    #  from /Users/user/Documents/Projects/ProjectX/Podfile:40
    #  -------------------------------------------
    #  ## Logging
    >  pod 'CocoaLumberjack', :git => 'https://github.com/CocoaLumberjack/CocoaLumberjack.git',  :branch => 'swift_2.0'
    #  
    #  -------------------------------------------

Given that the podfile and version of cocoa pods hasn't been changed, verified with version control, I am confused as to why Cocoapods suddenly believes there is a syntax error suddenly.

**Comment by orta** [comment]
This is _super_ weird, can you zip up the Podfile and put it up somewhere? That's legit ruby.

**Comment by segiddins** [comment]
What versions of CocoaPods and Ruby are you using?

**Comment by jwardle** [comment]
Right, I found what the issue was: Earlier in the file I had a comment that used // instead of # (habit adding // rather than #...) in the podfile. I have removed this comment, performed 'pod update' and it cleanly runs. The // comment was 9 lines above the location it was complaining about as mentioned in the OP.

VC shows the comment having been in place for quite a while - strange that its choked now rather than before...
